src/Sarif.Multitool.FunctionalTests/TestData/MessagesShouldEndWithPeriod/Invalid.sarif

```diff
@@ -110,48 +110,6 @@
               ]
             }
           ],
-          "graphs": {
```

Comment:
>graphs [](start = 11, length = 6)

You have removed all this test content because we decided not to author a transformer. Did you consider just manually updating all this content to conform to the new standard instead? shouldn't be hard.